test/import-tests.js

```diff
@@ -377,3 +377,10 @@ describe("Import from GraphML", function () {
     });
   });
 });
+
+describe('sheet-named-incorrectly', function () {
+  it('1 wrong name warning per file', function () {
```

Comment:
To follow the structure of the other warnings, this should likely be written as "should return one wrong name warning". Also, I think this test would fit a little better in errors-sheet-modification rather than import tests. Import tests is for SIF/ML.